test-e2e/config-import.js

```diff
@@ -64,6 +64,83 @@ test('config import - load and re-load config manually', async (t) => {
   )
 })
 
+test('deletion of data before loading a new config', async (t) => {
+  const manager = createManager('device0', t)
+  const project = await manager.getProject(await manager.createProject())
+
+  // load default config
+  await project.importConfig({
+    configPath: defaultConfigPath,
+  })
+  const nPresets = (await project.preset.getMany()).length
+  const nFields = (await project.field.getMany()).length
+  const nTranslations = (await project.$translation.dataType.getMany()).length
+
+  // load another config
+  await project.importConfig({
+    configPath: './tests/fixtures/config/validConfig.zip',
+  })
+
+  // load default config again
+  await project.importConfig({
+    configPath: defaultConfigPath,
+  })
+
+  assert.equal(
+    (await project.preset.getMany()).length,
+    nPresets,
+    'after loading config 1, then 2, then 1 again, number of presets should be equal'
+  )
+  assert.equal(
+    (await project.field.getMany()).length,
+    nFields,
+    'after loading config 1, then 2, then 1 again, number of fields should be equal'
+  )
+  assert.equal(
+    (await project.$translation.dataType.getMany()).length,
+    nTranslations,
+    'after loading config 1, then 2, then 1 again, number of translations should be equal'
+  )
+})
+
+test('failing on loading a second config should not delete any data', async (t) => {
+  const manager = createManager('device0', t)
+  const projectId = await manager.createProject()
+  const project = await manager.getProject(projectId)
+  // load default config
+  await project.importConfig({ configPath: defaultConfigPath })
+
+  const nPresets = (await project.preset.getMany()).length
+  const nFields = (await project.field.getMany()).length
+  const nTranslations = (await project.$translation.dataType.getMany()).length
+
+  // load wrong config
```

Comment:
Nit: this isn't just a _wrong_ config, it's a bogus path.

```suggestion
  // load a non-existent config
```